CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
 # Contributing
 
-Please _read_ before contributing to commercetools Documentation Kit in order to get familiar with the guidelines for contributing to the project.
+Please _read_ before contributing to commercetools Documentation Kit to get familiar with the guidelines for contributing to the project.
 
 ## Core Ideas
 
-We want to create documentation websites that are visually and functional consistent. To do so, we provide a set of tools and components that should be used across the different websites, for example a Gatsby documentation theme.
+At commercetools we want to create documentation websites that are visually and functional consistent. To do so, commercetools provide a set of tools and components that should be used across the different websites, for example a Gatsby documentation theme.
```

Comment:
@nkuehn actually can you clarify how we should write and refer to `commercetools`? Is it 3rd person singular or 1st person plural?

- [ ] `commercetools does`
- [ ] `commercetools do`

/cc @FreddieK

Reply:
Hi, this is not a commercetools specific question, it's a common grammar issue in english.  We generally bias towards American english and this is written and rather formal text, so a company is singular 3rd person ("it").  
Informally, in the UK and in spoken language "they" is possible too.  And in 1st person "we" it's okay too as in the sentence above ("at commercetools, we do XYZ").  But not "commercetools are going to the bar tonight", that's "commercetools is going to the bar tonight" in US English. 

To add to the confusion or clarification here's two decent articles:
 * https://erinwrightwriting.com/singular-or-plural-pronouns-for-a-company/ 
 * https://www.grammarphobia.com/blog/2018/01/it-or-they.html

/cc @adinakleine  in case you want to update our style guide